manifests/config.pp

```diff
@@ -178,6 +178,18 @@
       }
     }
     'systemd': {
+      # Puppet 5 doesn't have https://tickets.puppetlabs.com/browse/PUP-3483
+      # and camptocamp/systemd only creates this relationship when managing the service
+      if versioncmp($facts['puppetversion'],'6.1.0') < 0 {
+        exec { "prometheus_daemon_reload":
```

Comment:
camptocamp/systemd has a reload resource which we should use here, rather than duplicating it

Reply:
The whole point of this PR is to _not_ use `systemd::systemctl::daemon_reload` as it causes a dependency cycle, see https://github.com/voxpupuli/puppet-prometheus/issues/434 .

Reply:
I added comments to make this clear.